jdaviz/configs/cubeviz/plugins/parsers.py

```diff
@@ -8,9 +8,11 @@
 from astropy.nddata import StdDevUncertainty
 from astropy.time import Time
 from astropy.wcs import WCS
-from specutils import Spectrum1D
+from specutils import Spectrum1D, SpectralAxis
 
+from jdaviz.configs.imviz.plugins.parsers import prep_data_layer_as_dq
```

Comment:
So, circular import isn't a problem anymore?

Reply:
I'm not sure why this is in the diff? Was this removed recently?

Reply:
The diff says you moved the import up. 🤷‍♀️

Reply:
changed it back